src/metabase/api/slack.clj

```diff
@@ -9,16 +9,18 @@
 (defendpoint PUT "/settings"
   "Update Slack related settings. You must be a superuser to do this."
   [:as {{slack-token :slack-token, metabot-enabled :metabot-enabled, :as slack-settings} :body}]
-  {slack-token     [Required NonEmptyString]
```

Comment:
it should at least still be `slack-token NonEmptyString`. No longer `Required`, but if it is passed an non-nil, it should be a non-empty string.